prem_utils/models.json

```diff
@@ -429,6 +442,7 @@
             "provider": "azure",
             "models": [
                 {
+                    "tier": "high",
                     "slug": "gpt-4-32k-azure",
```

Comment:
Can we make also this model (high tier) not available on free tier as for cohere opus?